### PR TITLE
`Programming exercises`: Improve tooltips and exercise summary on the instructor details page

### PR DESCRIPTION
**Pull request by florian-glombik**
#### General
- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

#### Client
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context
Improve usability of the programming exercise creation process by adjusting translations and displayed information.

### Description
- Hiding build plan ids in exercise summary if the setup is using localVC
- Adding description tooltip for auxiliary repositories
- Fixing translations within auxiliary repository table
- Removing tooltip beside created auxiliary repositories

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Programming exercise on localVC setup

1. Check that the build plan ids are not displayed in the programming exercise details view, if the setup is using localVC.
2. In the programming exercise creation / edit page, check that the "Add auxiliary repository" button has a useful tooltip.
3. In the programming exercise creation / edit page, check that the auxiliary repository table has correct translations.
4. In the programming exercise creation / edit page, check that the tooltip beside the created auxiliary repositories are not displayed anymore

#### Code Review
- [ ] Code Review 1
- [ ] Code Review 2
#### Manual Tests
- [x] Test 1
- [ ] Test 2

### Screenshots
| Change                          | Before                                                                                                                    | Now                                                                                                 |
|---------------------------------|---------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
| Hide build plan ids             | image                        | image |
| Add Aux Repo tooltip            | image                        | image  |
| Fix Aux Repo table translations | image                        | image  |
| Removed Aux Repo tooltips       | image Tooltips not displayed on hover | image  |

### CODE REVIEW

File: src/main/webapp/i18n/en/programmingExercise.json [hunk 1]
```diff
@@ -462,8 +462,10 @@
             "auxiliaryRepository": {
                 "error": "There is a problem with the auxiliary repository.",
                 "addAuxiliaryRepository": "Add Auxiliary Repository",
+                "usageDescription": "You can use auxiliary repositories to provide additional Code that students cannot see or modify. The additional Code is inserted at the specified Checkout Directory during the build of the submission.",
```
Comment (pzdr7) [inline src/main/webapp/i18n/en/programmingExercise.json:465]: Can you double-check the capitalization here? (esp. `Code`)

**Review by pzdr7 — DISMISSED** [review]
Tested on TS2. Looks good 👍 

- No build plan IDs ✅ 
- Aux repo tooltip ✅ 
- Aux repo table ✅ 
- No tooltip for aux repo in preview ✅

File: src/main/webapp/i18n/en/programmingExercise.json (hunk 1 above)
Comment (florian-glombik) [reply src/main/webapp/i18n/en/programmingExercise.json]: I thought a bit about it, but stumbled across it aswell - it does not seem to be fully consistent, but `code` is probably the better way.

**Review by pzdr7 — APPROVED** [review]
👍

**Review by krusche — APPROVED** [review]

**Pull request merged**